### ISSUE TITLE
Error in gulp-sourcemaps and Less

### ISSUE THREAD

**Issue by rbatllet** [floridoo/gulp-sourcemaps#196]
I'm using `node v5.11.1` and `gulp-sourcemaps 1.6.0` and when I run this task:
```
gulp.task('less', function() {
  return gulp.src(appConfig.app + '/styles/main.less')
    .pipe(sourcemaps.init())
    .pipe(less({
      paths: ['.']
    }))
    .pipe(autoprefixer({
      browsers: ['last 2 versions'],
      cascade: false
    }))
    .pipe(sourcemaps.write('.'))
    .pipe(gulp.dest(appConfig.temp + '/styles'));
});
```
I get an `invalid mapping error`:

```
Potentially unhandled rejection [2] Error: Invalid mapping: {"generated":{"line":7024,"column":2},"source":"/Users/user/develop/my-project/app/styles/theme/landing.less","original":{"line":1,"column":null},"name":null}
    at SourceMapGenerator_validateMapping [as _validateMapping] (/Users/user/develop/my-project/node_modules/source-map/lib/source-map-generator.js:277:13)
    at SourceMapGenerator_addMapping [as addMapping] (/Users/user/develop/my-project/node_modules/source-map/lib/source-map-generator.js:101:12)
    at SourceMapOutput.add (/Users/user/develop/my-project/node_modules/less/lib/less/source-map-output.js:88:42)
    at Element.genCSS (/Users/user/develop/my-project/node_modules/less/lib/less/tree/element.js:42:12)
    at Selector.genCSS (/Users/user/develop/my-project/node_modules/less/lib/less/tree/selector.js:104:21)
    at Ruleset.genCSS (/Users/user/develop/my-project/node_modules/less/lib/less/tree/ruleset.js:389:21)
    at Directive.outputRuleset (/Users/user/develop/my-project/node_modules/less/lib/less/tree/directive.js:124:18)
    at Media.genCSS (/Users/user/develop/my-project/node_modules/less/lib/less/tree/media.js:34:10)
    at Ruleset.genCSS (/Users/user/develop/my-project/node_modules/less/lib/less/tree/ruleset.js:414:18)
    at SourceMapOutput.toCSS (/Users/user/develop/my-project/node_modules/less/lib/less/source-map-output.js:126:24)
    at SourceMapBuilder.toCSS (/Users/user/develop/my-project/node_modules/less/lib/less/source-map-builder.js:23:35)
    at ParseTree.toCSS (/Users/user/develop/my-project/node_modules/less/lib/less/parse-tree.js:33:47)
    at /Users/user/develop/my-project/node_modules/less/lib/less/render.js:31:40
    at /Users/user/develop/my-project/node_modules/less/lib/less/parse.js:63:17
    at Object.finish [as _finish] (/Users/user/develop/my-project/node_modules/less/lib/less/parser/parser.js:183:28)
    at Object.ImportVisitor._onSequencerEmpty (/Users/user/develop/my-project/node_modules/less/lib/less/visitors/import-visitor.js:35:14)
```

I'm using gulp-less 

```
$ npm ls less
MyProject@1.0.0 /Users/user/develop/my_project
└─┬ gulp-less@3.0.5
  └── less@2.7.0 
```

**Comment by mgol** [comment]
The same happens on Node.js `6.1.0` with `gulp-less` `3.1.0` & `gulp-sourcemaps` `1.6.0` on OS X 10.11.4.

**Comment by rbatllet** [comment]
I think the error is caused by Less v2.7.0 that it has been updated one day ago (https://github.com/less/less.js/releases/tag/v2.7.0)

**Comment by cecotw** [comment]
It looks like its breaking on the media queries.

**Comment by rbatllet** [comment]
Good news! 

It has been fixed in `Less 2.7.1`. It works great now!.

```
└─┬ gulp-less@3.1.0
  └── less@2.7.1 
```
 `node_modules` needs to be removed and then run  `npm install` again.
